packages/components/src/styles/form-styles.ts

```diff
@@ -72,15 +61,20 @@ export const getBaseChildStyles = (
       }),
       ...additionalDefaultJssStyle,
     },
-    ...(hoverMediaQuery({
-      // with the media query the selector has higher priority and overrides disabled styles
-      [`::slotted(${child}:not(:disabled):not(:focus):not([readonly]):hover)`]: {
-        borderColor: formStateHoverColor || primaryColor,
-        ...prefersColorSchemeDarkMediaQuery(theme, {
-          borderColor: formStateHoverColorDark || primaryColorDark,
-        }),
-      },
-    }) as Styles),
+    ...(!isLoading &&
+      (hoverMediaQuery({
+        // with the media query the selector has higher priority and overrides disabled styles
+        [[
+          `::slotted(${child}:not(:disabled):not(:focus):not([readonly]):hover)`,
+          `label:hover~* ::slotted(${child}:not(:disabled):not(:focus):not([readonly]))`,
+          'label:hover~* ::part(p-select-wrapper-dropdown)',
```

Comment:
This part should probably not be there for every component?
What about prefixing? And sharing the selector?
Also why create a string array and then join it?

Reply:
I'll add a condition for the select.
Prefixing is not needed. 
String Array can be refactored to string interpolation again.